### DISCUSSION TITLE
Resize the navbar

### DISCUSSION THREAD

**Discussion by awesome-llama** [ScratchAddons/ScratchAddons#2489] [Ideas] [answered]
The current navbar is a bit too large. 

Shrinking it down by changing the font size to around 20px and navbar height to 80px makes the site much more comfortable.

**Comment by mxmou** [comment, marked as answer]
I think you wanted to post this [here](https://github.com/FlagClicked/FlagClicked/discussions).

**Reply by awesome-llama** [reply]
You are right! I have no idea how I ended up posting here without realising.
